### ISSUE TITLE
"The Skyrim compiler path is not a valid file." 

### ISSUE THREAD

**Issue by hughrbeyer** [Gawdl3y/atom-build-papyrus#1]
I'm getting this error. The compiler path is

```
C:/games/steam/steamapps/common/Skyrim/Papyrus Compiler/PapyrusCompiler.exe
```

Set in both .build-papyrus.yml and in the plugin settings.

I know the path is correct because I copied it directly from the windows explorer. I tried with both backslashes and forward slashes--same result either way.

**Comment by hughrbeyer** [comment]
Found it. The settings override .build-papyrus.yml, surprise number one. And the settings require a full path include the exe, but the package ships with the default path which includes the directory only, which should be fixed.

**Comment by Gawdl3y** [comment]
The default path is correct, it includes the full path to the executable (`C:\Program Files (x86)\Steam\SteamApps\common\Skyrim\Papyrus Compiler\PapyrusCompiler.exe`).

The `compiler` setting in `.build-papyrus.yml` does in fact override the package settings when building, but the validation doesn't take it into account. I should fix that. ;)

**Comment by Gawdl3y** [comment]
Oh, hang on a minute - that notification is only for the package setting anyways. It will still reappear when the build provider is loading, though.

**Comment by Gawdl3y** [comment]
Fixed with @59f6c4f + @6810c34 (no clue how I managed to split these). Released in v1.4.3.

**Comment by hughrbeyer** [comment]
Incidentally, got it working and it's beautiful. So there's that.
